common/src/main/scala/com/gu/zuora/ZuoraService.scala

```diff
@@ -26,12 +28,26 @@ class ZuoraService(config: ZuoraConfig, client: FutureHttpClient, baseUrl: Optio
   def getAccount(accountNumber: String): Future[GetAccountResponse] =
     get[GetAccountResponse](s"accounts/$accountNumber")
 
+  def getAccountIds(identityId: String): Future[List[String]] = {
+    val queryData = QueryData(s"select AccountNumber from account where IdentityId__c = '$identityId'")
```

Comment:
This isn't SQL it's ZOQL which is quite a bit more constrained, but I'll parse it to a Long first to be safe.